Cargo.toml

```diff
@@ -59,9 +59,12 @@ prometheus = "0.13.4"
 
 # crypto
 blst = "0.3.11"
-tree_hash = "0.5"
-tree_hash_derive = "0.5"
+tree_hash = "0.6.0"
+tree_hash_derive = "0.6.0"
```

Comment:
Double check if updating to this newer version is okay (breaking changes).

Reply:
The necessity for this upgrade seems to have been eliminated during refactoring. Reverting as not immediately necessary. 👍